lib/js/components/SectionHeader/SectionHeader.vue

```diff
@@ -46,6 +46,7 @@ $icons-and-slot-min-height-l: 50px;
 	display: flex;
 	flex-wrap: wrap;
 	justify-content: space-between;
+	margin: $space-xxxxxs 0;
```

Comment:
why do you use `margin` on top level element? We usually set margin on a component, to set correct space between items - in such scenario this margin will be overriten. It's safer to use padding.

Reply:
Right, fixed.